packages/cli/__tests__/workflows/__system__/List/activeWorkflows/cli.workflows.list.activeWorkflows.system.test.ts

```diff
@@ -27,7 +27,7 @@ let wfKey: string;
 let system: string;
 let owner: string;
 let wfName: string;
-const workflow = join(__dirname, "../../../testfiles/demo.xml");
+const workflow = join(__dirname, "../../../../../../workflows/__tests__/__system__/testfiles/demo.xml");
```

Comment:
This is not redundant - This goes from the CLI package to the workflows package. Both the CLI and APIs use this demo XML file.

Reply:
Thanks for clarifying 🙂 While it's not ideal to have the packages depending on each other outside of top-level imports, I'm ok with it for now since it's limited to test files for the CLI package, not in any published code or across multiple SDK packages.